package.json

```diff
@@ -0,0 +1,10 @@
+{
+    "private": true,
+    "devDependencies": {
+        "bats": "^1.3.0",
+        "bats-assert": "^2.0.0"
+    },
+    "scripts": {
+        "test": "bazel test ..."
+    }
```

Comment:
I don't think we need to worry about enabling `npm test`. If we do, then it shouldn't just proxy to `bazel` since devs might not have it installed.

Reply:
This seems to be auto-generated. I will remove it.